sklearn/utils/_test_common/instance_generator.py

```diff
@@ -1017,6 +1017,24 @@ def _yield_instances_for_check(check, estimator_orig):
             "sample_weight is not equivalent to removing/repeating samples."
         ),
     },
+    MultiOutputClassifier: {
+        "check_param_validation": ("fit expects argument Y, test provides argument y"),
+        "check_sample_weight_equivalence_on_dense_data": (
+            "fit expects argument Y, test provides argument y"
+        ),
+        "check_estimator_sparse_array": (
+            "predict_proba returns a list of arrays, test expects array"
+        ),
+        "check_estimator_sparse_matrix": (
+            "predict_proba returns a list of arrays, test expects array"
+        ),
```

Comment:
didn't we have an issue on this @glemaitre ? I can only find https://github.com/scikit-learn/scikit-learn/issues/19880

Reply:
The one that I recall is this one. Basically it is to know if we return a compact from (an array) or a list of arrays.